doc/source/serve/deployment.rst

```diff
@@ -206,6 +206,102 @@ Please refer to the Kubernetes documentation for more information.
 Monitoring
 ==========
 
+Logging
+-------
+
+.. note::
+
+  For an general overview of logging in Ray, see `Ray Logging <../ray-logging.html>`__.
+
+When looking through log files of your Ray Serve application, it is useful to know which backend and replica each log line originated from.
+To automatically tag your logs with the current backend tag and replica tag (of the form ``backend_tag#<random letters>``), use the following function:
+
+.. autofunction:: ray.serve.get_backend_logger
+
+To write your own custom logger using Python's ``logging`` package, you may find the following two methods useful:
+
+.. autofunction:: ray.serve.get_current_backend_tag
+.. autofunction:: ray.serve.get_current_replica_tag
```

Comment:
I'd add a subsection here saying "configuring logger" or something, then another below that says "Loki Tutorial"